### DISCUSSION TITLE
Add flipjump language support

### DISCUSSION THREAD

**Discussion by tomhea** [compiler-explorer/compiler-explorer#6666] [Ideas]
Hi, I'm the creator of [flipjump](https://github.com/tomhea/flip-jump), and I thought about adding flipjump support to your compiler.
Is it something that will be merged if I'll do it properly? If so I might start working on it :)

Thanks, Tom.

**Comment by tomhea** [comment]
Wow, this discussion entry number is awesome.

**Comment by partouf** [comment]
I don't believe we have something in place to support language that require python. There have been some attempts, but nothing concrete yet. So it would not be easy for you to add, unfortunately. That situation might change, but there are a lot of other things on our to do list, so I'm not sure when - probably not this year.

**Reply by tomhea** [reply]
Alright, Thanks
